apps/services/sessions/src/app/sessions/sessions.service.ts

```diff
@@ -87,21 +92,24 @@ export class SessionsService {
   }
 
   create(session: CreateSessionDto): Promise<Session> {
-    const { id, sessionId, userAgent, ...rest } = session
+    const { id, sessionId, userAgent, ipLocation, ...rest } = session
 
     // Todo: Remove this when we have migrated IDS to use sessionId
     const sid = sessionId || id
 
     if (!sid) {
       throw new Error('Missing sessionId.')
     }
+    if (!ipLocation) {
+      this.logger.error('ipLocation missing, defaulting to Maxmind ip lookup')
```

Comment:
This should at most be `warning`. You're simply notifying that you're using a default instead of the preferred method/value.
```suggestion
      this.logger.warning('ipLocation missing, defaulting to Maxmind ip lookup')
```